src/shared/components/date-bubble/index.js

```diff
@@ -1,4 +1,5 @@
 // Displays date bubble
+/* eslint-disable max-len */
```

Comment:
I think it would be better to fix the error than to disable the rule.

Also this component looks like a better fit for a function rather than a class.